anautilities.py

```diff
@@ -57,26 +69,18 @@ def rejectOutliersMADOneSided(arrayData, thresh=3.5, rejectHighTail=True):
 #Use inter-quartile range (IQR) to reject outliers
 #Returns a boolean array with True if points are outliers and False otherwise.
 def isOutlierIQR(arrayData):
-    if len(arrayData.shape) == 1:
-        arrayData = arrayData[:,None]
-        pass
-
```

Comment:
Are you sure this works as expected with these lines removed?  I would caution against this as during testing I found without these lines unexpected behavior occurred.